src/main/java/car/Game.java

```diff
@@ -0,0 +1,125 @@
+package car;
+
+import java.util.*;
+
+public class Game {
+
+    public String carNameinput() {
+        System.out.println("경주할 자동차 이름을 입력하세요(이름은 쉼표(\\,)를 기준으로 구분)");
+        Scanner scanner = new Scanner(System.in);
+        return scanner.nextLine();
+    }
+
+    public String countInput() {
+        System.out.println("시도할 횟수는 몇 회인가요?");
+        Scanner scanner = new Scanner(System.in);
+        return scanner.nextLine();
+    }
+
+    public String[] splitBlank(String str) {
+        return str.split(",");
+    }
+
+    public void excute() {
+
+        String carInput = carNameinput();
+
+        String[] carInputs = splitBlank(carInput);
+
+        List<Car> cars = new ArrayList<Car>();
+
+        for (int i = 0; i < carInputs.length; i++) {
+            // 이름 잘라주는거
+            cars.add(getCar(carInputs[i]));
+        }
+
+        if (cars.size() < 2)
+            return;
+
+        String count = countInput();
+
+        System.out.println("실행 결과");
+        carRacing(toInt(count), cars);
+    }
+
+    public Integer toInt(String number) {
+        return Integer.valueOf(number);
+    }
+
+    public Car getCar(String carName) {
+        int length = carName.length();
+        if(length > 5)
+            length = 5;
+        Car c = new Car();
+        carName = carName.substring(0, length);
+        c.setName(carName);
+        return c;
+    }
+
+    public void carRacing(int count, List<Car> cars) {
+
+        int nowCount = 0;
+
+        while (nowCount < count) {
+            carDrive(cars);
+
+            printCars(cars);
+
+            nowCount++;
+        }
+
+        printWinner(cars);
+    }
+
+    public int getMoveValue() {
+        int randomNumber = (int) (Math.random() * 10);
+        if (randomNumber >= 4) return 1;
+        return 0;
+    }
+
+    public void carDrive(List<Car> cars) {
+        for (int i = 0; i < cars.size(); i++) {
+            int nowKm = cars.get(i).getKm();
+            cars.get(i).setKm(nowKm + getMoveValue());
+        }
+    }
+
+    public String printLines(int km) {
+        String lines = "";
+        for (int i = 0; i < km; i++) {
+            lines = lines + "-";
```

Comment:
```suggestion
            lines += "-";
```
증감연사자로도 변경해볼 수 있을 것 같아요 😉